OneSchedule/appsettings.json

```diff
@@ -6,5 +6,13 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+
+  "BotToken": "1774119603:AAFCWMV12zS0SLBx4kC3A-suLJ511wP4oOo",
```

Comment:
Create class for Telegram settings because in future we can have more values there
Don't push secret values to git. This value should be kept in private, consider configuring _user secrets_